### DISCUSSION TITLE
Create only bullet in the section

### DISCUSSION THREAD

**Discussion by titipata** [posquit0/Awesome-CV#603] [Q&A]
Hello @posquit0 and others, I just want to create a section with all bullet point in it, without creating `cventry`. Is there a way that I can use `cvitems` directly?

Thanks!

**Comment by posquit0** [comment]
Oh... sorry, I have not finished this work yet after reading your email. I'm considering to rewrite some codes in `awesome-cv.cls` to support it.

If someone has temporary solutions, please comment on this thread

**Comment by titipata** [comment]
Yeah, thanks so much! In this case, I want to put it in extracurricular section and/or skills section which tends to be multiple bullets. It would be nice if I can simply write

``` latex
\begin{cvitems}
  \item ...
  \item ...
\end{cvitems}
```

**Comment by bluenex** [comment]
Although it's not as easy as

``` tex
\begin{cvitems}
  \item ...
  \item ...
\end{cvitems}
```

but I came up with this workaround, just leave `\cventry` arguments blank and add `\vspace{-5mm}`. The rendered should be good now.

``` tex
\begin{cventries}
\vspace{-5mm}
  \cventry
    {}
    {}
    {}
    {}
    {
      \begin{cvitems}
        \item {}
        \item {}
        \item {}
      \end{cvitems}
    }
\end{cventries}
```

**Comment by titipata** [comment]
+1 for @bluenex. This solves my issue for now. :dancers:

**Comment by dcsim** [comment]
Would pulling the skills section from the cv/ work for you?

**Comment by fujunswufe** [comment]
@posquit0 Hello and others, can we skip cvitems and skip this line, so we can save some space

**Comment by gandhishripala** [comment]
Hi all,
I tried to fix this issue and came up with solution. Just as @bluenex has suggested, I used \cventry, but I have modified it a bit.
Add following code in awesome-cv.cls file.
```
\newenvironment{expentries}{%
  \vspace{\acvSectionContentTopSkip}
  \begin{center}
}{%
  \end{center}
}
% Usage: \expentry{<description>}
\newcommand*{\expentry}[1]{%
  \vspace{-2.0mm}
  \setlength\tabcolsep{0pt}
  \setlength{\extrarowheight}{0pt}
  \begin{tabular*}{\textwidth}{@{\extracolsep{\fill}} L{\textwidth - 4.5cm} R{4.5cm}}
    \multicolumn{2}{L{\textwidth}}{\descriptionstyle{#1}}
  \end{tabular*}%
}
```
It can be used in tex files as:
```
\begin{expentries}

%------------------------------------------------

\expentry
{ % Description(s) bullet points
\begin{cvitems}
\item {Bullet 1.}
\item {Bullet 2.}
\item {Bullet 3.}
\item {Bullet 4.}
\end{cvitems}
}

%------------------------------------------------

\end{expentries}
```

Although this might not be a optimal solution, requesting to provide feedback and suggestions on this, as I am new to LaTeX and would like to contribute and learn more.

Thank you.

**Comment by dcsim** [comment]
> Would pulling the skills section from the cv/ work for you?

```
% Define a line of cv information(skill)
% Usage: \cvskill{<type>}{<skillset>}
\newcommand*{\cvskill}[2]{%
      \ifstrempty{#2} % If empty 2
          {\multicolumn{2}{L{\textwidth}}{\descriptionstyle{#1}} \\} % cvitem bullets
          {\skilltypestyle{#1} & \skillsetstyle{#2} \\} % else existing skills format
}
```

This would add an alternative use of cvskill that creates bulleted lists and matches the formatting of cvitems. It looks a bit odd but you can use empty `#2`s and the existing skills at the same in a single cvskills environment if you want.

Would it be easier to do this and document its usage in the existing `skills.tex` example?
